archetypes/pom.xml

```
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.infinispan</groupId>
+        <artifactId>infinispan-parent</artifactId>
+        <version>11.0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <groupId>org.infinispan.archetypes</groupId>
+    <artifactId>infinispan-archetypes</artifactId>
+    <version>11.0.1-SNAPSHOT</version>
+
+    <name>Infinispan Archetypes</name>
+    <description>Archetypes for the Infinispan project</description>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>embedded</module>
+    </modules>
+
+    <properties>
+        <maven.archetype.version>2.3</maven.archetype.version>
```

can you try with the version used by `jboss-parent`?
```suggestion
        <maven.archetype.version>${version.archetype.plugin}</maven.archetype.version>
```

---

Done